src/main/java/mu/semte/ch/api/kalliope/rest/AppController.java

```diff
@@ -42,7 +42,15 @@ public AppController(PersistService persistService) {
     
     return ResponseEntity.ok().build();
   }
+
+  @PostMapping("/bulk")
+  public ResponseEntity<Void> bulk(@RequestBody String dataModel) {
+    persistService.writeBulk(ModelUtils.toModel(dataModel, Lang.TURTLE.getName()));
+    
+    return ResponseEntity.ok().build();
```

Comment:
you could use ResponseEntity.accepted().build when the the service is asynchronous 😃  Not blocking, lgtm